dinosure_course_gordon_fleming/code/06-scheduled-functions.js

```diff
@@ -1 +1,37 @@
 // 06-scheduled-functions
+/**
+ * Executed on the schedule defined in `.root-config.json`.
+ * @param {object} params
+ * @param {PlatformPolicy} params.policy The policy for which the scheduled function is running.
+ * @param {PlatformPolicyholder} params.policyholder The policyholder linked to the policy
+ * @return {ProductModuleAction[] | void} The actions to be queued by the platform.
+ */
+const applyAnnualIncrease = ({ policy, policyholder }) => {
+  // Check if the policy is older than a year
+  const policyStartDate = moment(policy.start_date);
+  const today = policy.module.today ?? moment();
```

Comment:
What is the point of looking/assigning `policy.module.today`?

Reply:
This is to check if today was the 1st of January, would the policy automated change be activated.

Reply:
Ah I see, but the function will only run on the 01/01 so it seems redundant to check it here again.

Reply:
In testing you assign it, sure. But in force, production policy won't ever have this value assigned so it will always be `undefined`. Your return action doesn't change the module data so temporarily mutating to an object is bad practice because you'll never use it as such. 

If you had to check, you could simply to this without mutating the object:

```javascript
if (moment().format(MM-DD) === '01-01') {
  // do the thing
}
```

Reply:
Okay great thank you, I understand now.